### PR TITLE
London class 8/teniola ogunjinrin

### PR DESCRIPTION
**Pull request by teniolao**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:Teniola Ogunjinrin
- Your City: London
- Your Slack Name: Teniola O

# Homework Details

- Module: JAVASCRIPT
- Week: 1

# Notes

- What did you find easy? doing the exercises helped me grasp the concepts

- What did you find hard? it took me a while to understand how to navigate to files using the terminal. I think i was doing it wrong at first . because i completed my exercise when i did a pull request only 1 exercise saved so i had to start all over again

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: exercises/I-floats/exercise.js [hunk 1]
```diff
@@ -1,2 +1,17 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let totalNumber = numberOfStudents + numberOfMentors;
+
+let quotientOfStudents = numberOfStudents / totalNumber;
+let quotientOfMentors = numberOfMentors / totalNumber;
+
+let productOfStudents = quotientOfStudents * 100;
```
Comment (mandeepsangha) [inline exercises/I-floats/exercise.js:8]: could we have added the " * 100"  to line 5 to save creating an extra variable or alternatively add it into the console.log on line 14 : something like this: math.round(quotientOfStudents*100) + "%"

File: exercises/L-functions-nested/exercise.js [hunk 2]
```diff
@@ -3,3 +3,9 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function greeting (){
+    return Hello;
+}
+
+console.log (greeting + mentor1)
```
Comment (mandeepsangha) [inline exercises/L-functions-nested/exercise.js:11]: we can apply .toUpperCase to make the string all capital letters

**Review by mandeepsangha — COMMENTED** [review]
Hi Teniola Great job with these exercises, i like how you breakdown the problems and your naming of the variables you use is very clear :+1: . If you have time next week I would recommend reviewing the Extra tasks they are really interesting and thankfully there is only 3 of them :+1:

File: exercises/I-floats/exercise.js (hunk 1 above)
Comment (teniolao) [reply exercises/I-floats/exercise.js:8]: thanks Mandeep i was able to take out four lines of code

**Review by DelroyGayle — COMMENTED** [review]
Dear Teniola

Excellent presentation of code.
Great example to myself and others.
Thank you

**Pull request closed without merging**